templates/master/00-master/on-prem/files/keepalived-keepalived.yaml

```diff
@@ -54,9 +54,10 @@ contents:
         fall 2
     }
 
+    {{`{{ range $i, $config := .Configs }}`}}
     {{`{{$nonVirtualIP := .NonVirtualIP}}`}}
 
-    vrrp_instance {{`{{ .Cluster.Name }}`}}_API {
+    vrrp_instance {{`{{ .Cluster.Name }}`}}_API_{{`{{$i}}`}} {
```

Comment:
I still don't understand how we get different configs for the VIPs :sweat_smile:

Reply:
See above. :-)